packages/react-components/react-progress/src/components/ProgressBar/useProgressBar.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { getNativeElementProps, resolveShorthand } from '@fluentui/react-utilities';
 import type { ProgressBarProps, ProgressBarState } from './ProgressBar.types';
+import { clampValue, clampMax } from '../../utils/index';
```

Comment:
_nit:_ Keep `type` imports at the end of the import statements

```suggestion
import { clampValue, clampMax } from '../../utils/index';
import type { ProgressBarProps, ProgressBarState } from './ProgressBar.types';
```